lib/cdo/rack/response.rb

```diff
@@ -0,0 +1,19 @@
+# Patch Rack::Response with extra utility methods.
```

Comment:
Monkey-patching is to be avoided when possible; is it essential to use it in this case?  

Could we for example just have a ResponseUtils class with `has_header?(response, key)`?